explainers/hidden-content-explainer.md

```diff
@@ -12,28 +12,21 @@ This is an explainer for two closely related features:
         hidden from the user (similar to `content-visibility: hidden`), but the
         content remains searchable via user-agent find-in-page algorithms.
 
-2. The `beforematch` event allows developers to display content to the user in
-    response to actions which scroll the page to a target element. It is fired
-    at render timing just before these actions scroll the page. The applicable
-    actions are the following:
+2. The `beforematch` event allows developers to display `hidden-matchable`
+    content to the user in response to searches which scroll the page to a
+    target element. It is fired at render timing before these actions scroll
+    the page. The applicable actions are the following:
     * There is a new find-in-page (ctrl+f) active match, in which case the event
       is fired with the matched element as the target.
-    * There is an element fragment navigation (`example.com/#foo`) or the fragment
-      is changed by script (`window.location.hash = ...`), in which case the event
-      is fired with the matched element as the target.
     * There is a [scroll-to-text](https://github.com/WICG/ScrollToTextFragment)
       navigation (`example.com/#:~:text=foo`), in which case the event is fired with
       the matched element as the target.
 
-Note that the 'matched element' in this document refers to one of the following:
-* In the find-in-page and scroll-to-text cases: The closest block-level ancestor
-  element in the flat tree containing the matching text.
-  If the match spans multiple block-level elements, then the first block-level
-  element is used.
-  Since the flat tree is used to determine the element to fire the event on,
-  shadow boundaries have no impact.
-* In the element fragment case: The element which was selected by id from the
-  fragment.
+Note that the 'matched element' in this document refers to the closest
+block-level ancestor element in the flat tree containing the matching text.
```

Comment:
Instead of matching text, say "match" and link to https://html.spec.whatwg.org/multipage/interaction.html#fip-matches

Reply:
I removed this text and put a link to fip-matches in the find-in-page bullet point. how does it look?

Reply:
Looks good